src/loader/README.md

```diff
@@ -2,20 +2,77 @@
 
 The loader has two main purposes:
 
-1) Load the correct versioned library for each database.
-Multiple databases in the same Postgres instance may contain
-different versions of TimescaleDB installed. The loader is
-responsible for loading the shared library corresponding
-to the correct TimescaleDB version for the database as soon
-as possible. For example, a database containing TimescaleDB
-version 0.8.0 will have timescaledb-0.8.0.so loaded.
+1) Load the correct versioned library for each database. Multiple databases in
+   the same Postgres instance may have different versions of TimescaleDB
+   installed. The loader is responsible for loading the shared library
+   corresponding to the correct TimescaleDB version for the database as soon as
+   possible. For example, a database containing TimescaleDB version 0.8.0 will
+   have timescaledb-0.8.0.so loaded.
 
-2) Starting background worker schedulers for each database.
-   Background worker schedulers launch background worker tasks
-   for TimescaleDB. The launcher is responsible for launching
-   schedulers for any database that has TimescaleDB installed.
-   This is done by a background task called the launcher.
+2) Starting background worker schedulers for each database. Background worker
+   schedulers launch background worker tasks for TimescaleDB. The launcher is
+   responsible for launching schedulers for any database that has TimescaleDB
```

Comment:
slightly misleading as schedulers are launched for all databases. Maybe say something like: 

`
The launcher is responsible for launching schedulers (one for each database) that are responsible for checking whether the TimescaleDB extension is installed in a database. If a scheduler finds an extension, its task is to schedule jobs for that database. In case of no TimescaleDB extension, the scheduler exits until it is reactivated for that database, which happens, for instance, when the extension is installed.
`